platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx

```diff
@@ -5,8 +5,14 @@ date: "12 Jun 2024"
 tags: [docker, compose, kubernetes, data, studios, deployment]
 ---
 
+You can enable Data Studios as part of your Seqera Platform instance.
+
 ## DNS configuration
 
+Each data studio is reachable at a unique URL, which includes a randomly generated subdomain name. For example, a data studio URL can be: `https://abcd.example.com/`, where `example.com` is your Seqera base domain name.
+
+To allow for uniquely generated third level domains, you must provide a wildcard TLS certificate. A wildcard certificate is one where the common name includes `*.` in the domain name, such as `*.example.com`.
+
 The proxy domain (`CONNECT_PROXY_URL`) has to be a subdomain of main platform domain (example: if the platform domain is
 `tower.io`, then the proxy domain should be `example.tower.io`).
```

Comment:
Please align your examples by sticking with a single common domain. You've got two different domain examples in two successive paragraphs.

Reply:
@gwright99, yep, this isn't complete, as you've gathered. This is still a combination of the original PR's content I didn't write and my best interpretive rewrite of what it might become, still incomplete.